### ISSUE TITLE
Crash if RichTextLabel line_separation constant is set below 0 in inspector

### ISSUE THREAD

**Issue by Spartan322** [godotengine/godot#64194]
### Godot version

v4.0.alpha13.official [59dcddc45]

### System information

archlinux 5.18.9-zen1-1-zen x86_64 Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere Micro-Star International Co., Ltd. [MSI] Radeon RX 580 Armor 4G OC amdgpu

### Issue description

If you set RichTextLabel's line_separation constant to below 0, it will crash the editor, strangely the display may still work with that value if you drag it below 0, but once it gets permanently set in the inspect it will still crash.

### Steps to reproduce

1. Create Project
2. Make a RichTextLabel scene
3. Go down to theme override constants
4. Tick line_separation
5. Click the down arrow once 

### Minimal reproduction project

The RichTextLabel is already prepared, just click the down arrow for the line_separation constant:
[RichTextLabelCrash.zip](https://github.com/godotengine/godot/files/9295570/RichTextLabelCrash.zip)

**Comment by timothyqiu** [comment]
```
ERROR: Index p_index = 1 is out of bounds (count = 1).
   at: operator[] (./core/templates/local_vector.h:168)
```

Backtrace:

```
(gdb) bt
#0  0x000055556016bbf5 in LocalVector<RichTextLabel::Line, unsigned int, false, false>::operator[] (this=0x6100000beb88, p_index=1) at ./core/templates/local_vector.h:168
#1  0x0000555560113fc7 in RichTextLabel::_notification (this=0x61d002b57a90, p_what=30) at scene/gui/rich_text_label.cpp:1782
#2  0x0000555560168284 in RichTextLabel::_notificationv (this=0x61d002b57a90, p_notification=30, p_reversed=false) at scene/gui/rich_text_label.h:40
#3  0x00005555648170d8 in Object::notification (this=0x61d002b57a90, p_notification=30, p_reversed=false) at core/object/object.cpp:735
#4  0x000055555f95931d in CanvasItem::_update_callback (this=0x61d002b57a90) at scene/main/canvas_item.cpp:135
#5  0x0000555559d60f02 in call_with_variant_args_helper<__UnexistingClass>(__UnexistingClass*, void (__UnexistingClass::*)(), Variant const**, Callable::CallError&, IndexSequence<>) (p_instance=0x61d002b57a90, 
    p_method=<error reading variable: Cannot access memory at address 0xffffffffffffffc0>, p_args=0x7fffffffd200, r_error=...) at ./core/variant/binder_common.h:262
#6  0x0000555559d5cfd5 in call_with_variant_args_dv<__UnexistingClass> (p_instance=0x61d002b57a90, p_method=(void (__UnexistingClass::*)(__UnexistingClass * const)) 0x55555f9591ce <CanvasItem::_update_callback()>, p_args=0x0, 
    p_argcount=0, r_error=..., default_values=...) at ./core/variant/binder_common.h:409
#7  0x0000555559d548a4 in MethodBindT<>::call(Object*, Variant const**, int, Callable::CallError&) (this=0x60c000129650, p_object=0x61d002b57a90, p_args=0x0, p_arg_count=0, r_error=...) at ./core/object/method_bind.h:320
#8  0x0000555564816716 in Object::callp (this=0x61d002b57a90, p_method=..., p_args=0x0, p_argcount=0, r_error=...) at core/object/object.cpp:678
#9  0x00005555640a571b in Callable::callp (this=0x7fffed996870, p_arguments=0x0, p_argcount=0, r_return_value=..., r_call_error=...) at core/variant/callable.cpp:62
#10 0x0000555564806c56 in MessageQueue::_call_function (this=0x6070000d8930, p_callable=..., p_args=0x7fffed996888, p_argcount=0, p_show_error=false) at core/object/message_queue.cpp:229
#11 0x00005555648074c0 in MessageQueue::flush (this=0x6070000d8930) at core/object/message_queue.cpp:275
#12 0x000055555fad77a1 in SceneTree::physics_process (this=0x619000314790, p_time=0.016666666666666666) at scene/main/scene_tree.cpp:420
#13 0x0000555559072382 in Main::iteration () at main/main.cpp:2855
#14 0x0000555558fa768d in OS_LinuxBSD::run (this=0x7fffffffe0c0) at platform/linuxbsd/os_linuxbsd.cpp:540
#15 0x0000555558f9affe in main (argc=4, argv=0x7fffffffe6a8) at platform/linuxbsd/godot_linuxbsd.cpp:72
```

**Comment by KoBeWi** [comment]
Doesn't seem to crash anymore. Can someone confirm?

**Comment by akien-mga** [comment]
Fixed by #65735.
